importer/tag/src/test/java/org/itsallcode/openfasttrace/importer/tag/TestTagImporter.java

```diff
@@ -252,14 +314,21 @@ private static SpecificationItemId id(final String artifactType, final String na
 
     private static SpecificationItem item(final String artifactType, final int lineNumber,
             final int counter, final SpecificationItemId coveredId)
+    {
+        return item(artifactType, lineNumber, counter, coveredId, emptyList());
+    }
+
+    private static SpecificationItem item(final String artifactType, final int lineNumber,
+            final int counter, final SpecificationItemId coveredId, List<String> neededArtifactTypes)
     {
         final SpecificationItemId generatedId = SpecificationItemId.createId(artifactType,
                 generateName(coveredId, lineNumber, counter), 0);
-        return SpecificationItem.builder() //
+        final Builder itemBuilder = SpecificationItem.builder() //
                 .id(generatedId) //
                 .addCoveredId(coveredId) //
-                .location(FILENAME, lineNumber) //
-                .build();
+                .location(FILENAME, lineNumber);
+        neededArtifactTypes.forEach(itemBuilder::addNeedsArtifactType);
```

Comment:
I personally found this notation always hard to read. It's a matter of taste though.